product-web-app/middleware/auth.ts

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios';
+import { NextFunction, Request, Response } from 'express';
+import jwt, { JwtPayload } from 'jsonwebtoken';
+
+export default class AuthMiddleware {
+  async authorisation(req: Request, res: Response, next: NextFunction) {
+    // console.log("***********************************************")
+    // console.log(req.path)
+    // console.log(req.path == '/auth/login')
+    if (req.path === '/auth/login' || req.path === '/auth/register') {
+      return next();
+    }
+
+    const secret = 'NOT_HARDCODED_SECRET';
```

Comment:
please put it in the class but outside of the method - it will be easier to find - and it should be taken from ENV variable so that we can change it in AWS